avocado/utils/path.py

```diff
@@ -191,3 +192,21 @@ def usable_ro_dir(directory):
             pass
 
     return False
+
+
+def sync_attempt():
+    """
+    Attempts (best-effort) to synchronize file buffers into disk
+
+    This method attempts to to call the native Python standard library
+    method where available, and then falls back to a ctypes wrapper when
+    necessary.
+    """
+    if hasattr(os, 'sync'):
+        os.sync()
```

Comment:
We already rely on `os.sync()` in `jobdata` module. If there's a problem with it's availability, we have to address that creating the fallback everywhere (i.e. https://github.com/avocado-framework/avocado/blob/master/avocado/core/jobdata.py#L86)

Reply:
`fsync()` is available on all Python versions, it's `sync()` that's not.

Reply:
ops :)
then it looks good.